docs/api-docs/storefront/graphql/graphql-api-overview.mdx

```diff
@@ -194,60 +189,76 @@ Content-Type: application/json
+
+Unlike typical GraphQL Storefront API tokens, Customer impersonation tokens _are_ sensitive, and should therefore **never** be exposed publicly, for example, to JavaScript or HTML. These tokens should not be used for front-end requests. They should be treated with the same care as other application secrets, just as you might treat an OAuth access token for BigCommerce's REST APIs. Requests using these tokens originating from a web browser will be rejected.
 
-Customer impersonation tokens should **never** be exposed publicly, for example, to JavaScript or HTML. These tokens should not be used for front-end requests. Unlike normal GraphQL Storefront API tokens, they are sensitive and should be treated like secrets, just as you might treat an OAuth access token for BigCommerce's REST APIs. Attempts to run requests using these tokens from a web browser will be rejected.
+It is not necessary to generate a new token for each customer ID. You may use a single token at any given time for your application, and specify the customer ID on a per-request basis.
 
 Consider this sample request using a Customer Impersonation token to run a request in the context of customer ID `123`.
 
 ```shell filename="Example request: Query as customer 123" showLineNumbers copy
-curl 'https://store.com/graphql' -H 'Authorization: Bearer TOKEN_GOES_HERE' -H 'X-Bc-Customer-Id: 123' --data-binary '{"query":"query CustomerInformation {\n  customer {\n    firstName\n    lastName\n    email\n  }\n}"}'
+curl 'https://store.com/graphql' -H 'authorization: Bearer TOKEN_GOES_HERE' -H 'x-bc-customer-id: 123' --data-binary '{"query":"query CustomerInformation {\n  customer {\n    firstName\n    lastName\n    email\n  }\n}"}'
 ```
 
+<Callout type="info">
+  * If your token were to become compromised and you wish to severe the Storefront API connection for your token, you can use the [Revoke a token](/docs/storefront-auth/tokens#revoke-a-token) endpoint. Only use this in emergency situations, and do not revoke tokens unnecessarily. Instead, use a shorter expiration and allow them to expire naturally.
+</Callout>
+
 ### Customer login
 
-If you're using the GraphQL Storefront API from a browser, for example, on top of your Stencil storefront, you can use the new Customer Login mutation to sign in to a customer account with an email address and a password. For server-side integrations, consider a customer impersonation token instead. This will set a session cookie in the browser which will authenticate the customer account on future requests:
+If you're using the GraphQL Storefront API from a browser, for example, on top of your Stencil storefront, you can use the Customer Login mutation to sign in to a customer account with an email address and a password. This will set a session cookie in the browser which will authenticate the customer account on future requests:
```

Comment:
```suggestion
If you're using the GraphQL Storefront API from a browser, for example, on top of your Stencil storefront, you can use the Customer Login mutation to sign in to a customer account with an email address and a password. This will set a session cookie in the browser, which will authenticate the customer account on future requests:
```